src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -19,29 +19,40 @@ import UIKit
 import UserNotifications
 import ExposureNotification
 
-enum OnboardingPageType: Int, CaseIterable {
-	case togetherAgainstCoronaPage = 0
-	case privacyPage = 1
-	case enableLoggingOfContactsPage = 2
-	case howDoesDataExchangeWorkPage = 3
-	case alwaysStayInformedPage = 4
-
-	func next() -> OnboardingPageType? {
-		OnboardingPageType(rawValue: rawValue + 1)
-	}
 
-	func isLast() -> Bool {
-		(self == OnboardingPageType.allCases.last)
-	}
-}
+final class OnboardingInfoViewController: UIViewController {
 
-extension OnboardingInfoViewController: RequiresAppDependencies {
+	@IBOutlet var imageView: UIImageView!
+	@IBOutlet var stateHeaderLabel: ENALabel!
+	@IBOutlet var stateTitleLabel: ENALabel!
+	@IBOutlet var stateStateLabel: ENALabel!
+	@IBOutlet var titleLabel: UILabel!
+	@IBOutlet var boldLabel: UILabel!
+	@IBOutlet var textLabel: UILabel!
+	@IBOutlet var linkTextView: UITextView!
+	@IBOutlet var nextButton: ENAButton!
+	@IBOutlet var ignoreButton: ENAButton!
 
-}
+	@IBOutlet var scrollView: UIScrollView!
+	@IBOutlet var stackView: UIStackView!
+	@IBOutlet var stateView: UIView!
+	@IBOutlet var innerStackView: UIStackView!
+	@IBOutlet var footerView: UIView!
 
-final class OnboardingInfoViewController: UIViewController {
-	// MARK: Creating a Onboarding View Controller
+	private var pageType: OnboardingPageType
```

Comment:
These should go into the `// MARK: - Private` section

Reply:
moved private vars to privat section